sfepy/discrete/fem/fields_base.py

```diff
@@ -1071,21 +1071,38 @@ def get_coor(self, nods=None):
         else:
             return self.coors[nods]
 
-    def get_connectivity(self, region, integration, trace_region=None):
-        """
-        Convenience alias to `Field.get_econn()`, that is used in some terms.
-        """
-        return self.get_econn(integration, region, trace_region=trace_region)
-
     def get_econn(self, conn_type, region, trace_region=None, local=False):
         """
         Get extended connectivity of the given type in the given region.
+
+        Parameters
+        ----------
+        conn_type: tuple or string
+            DOF connectivity type, eg. ('cell', 3) or 'cell'.
+            If the topological dimension not specified, taken from region.tdim.
+        region: sfepy.discrete.common.region.Region
+            The region for which the connectivity is required.
+        trace_region: None or string
+            If not None, return mirror connectivity according to `local`.
+        local: bool
+            If True, return local connectivity w.r.t. surface nodes,
+            otherwise return global connectivity w.r.t. all mesh nodes.
+
+        Returns
+        -------
+        econn: numpy.ndarray
+            Connectivity information.
```

Comment:
The extended connectivity array.

econn = extended conn = connectivity with higher order nodes - mention this somewhere?

Reply:
> econn = extended conn = connectivity with higher order nodes - mention this somewhere?

Any idea where?